pyaedt/generic/ibis_reader.py

```diff
@@ -0,0 +1,739 @@
+import os
+
+import pyaedt
+
+
+class Component:
+    """Component extracted from ibis model."""
+
+    def __init__(self):
+        self._name = None
+        self._manufacturer = None
+        self._pins = {}
+
+    @property
+    def name(self):
+        """Name of the component."""
```

Comment:
Consider adding examples to these properties.

Reply:
I did and it highlighted the fact that some tests were missing.